bump.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
```

Comment:
Please add a comment explaining the purpose of the script and how it is supposed to be called here for teammates to get a quick understanding of this script.

Reply:
You got it -- ill address next pass on this PR